src/clarity-angular/popover/signpost/signpost.ts

```diff
@@ -0,0 +1,347 @@
+import {
+    Component,
+    Input,
+} from "@angular/core";
+
+import { Point, PopoverOptions } from "../common/popover";
+import { IfOpenService } from "./if-open.service";
+import { Subscription } from "rxjs/Subscription";
+
+//aka where the arrow / pointer is at in relation to the anchor
+const signpostPositions: string[] = [
+    "top-left",
+    "top-middle",
+    "top-right",
+    "right-top",
+    "right-middle", // default
+    "right-bottom",
+    "bottom-right",
+    "bottom-middle",
+    "bottom-left",
+    "left-bottom",
+    "left-middle",
+    "left-top",
+];
+
+@Component({
+    selector: "clr-signpost",
+    template: `
+        <button
+            #anchor
+            type="button"
+            class="signpost-action btn btn-small btn-link"
+            [ngClass]="{active: open}"
+            clrSignpostToggle>
+            <clr-icon shape="info"></clr-icon>
+        </button> <!-- <==turn into ng-content to handle default signpost icon and user input. -->
```

Comment:
I don't remember if this `<ng-content>` feature was part of the initial MVC or not. I'm fine pushing it for later, we just need to make sure we open an issue for it.